Unit test is unstable

**Is this a BUG REPORT or FEATURE REQUEST?**:
BUG
**What happened?**:
Run ```./scripts/run-build.sh``` to build acr-builder. Unit test sometimes failed as the following error.

--- FAIL: TestCompileHappy (0.07s)
	assertions.go:247: 
	Error Trace:	build_test.go:314
			build_test.go:294	
			build_test.go:299
			build.go:20
			build.go:278
			workflow.go:70
			workflow.go:49
			build_test.go:284
			build_test.go:234
	Error:      	Max difference between 2018-02-01 09:14:03.003017782 +0000 UTC m=+0.0111730
13 and 2018-02-01 09:14:02 +0000 UTC allowed is 1s, but difference was 1.003017782s

**What did you expect to happen?**:
unit test passed

**How do you reproduce it (as minimally and precisely as possible)?**:

**Anything else we need to know?**:

**Environment**:

--- a/pkg/driver/build_test.go
+++ b/pkg/driver/build_test.go
@@ -311,7 +311,7 @@ func assertSameContext(t *testing.T, expected []string, actual *build.BuilderCon
 			timeStampFound = true
 			buildTime, err := time.Parse(constants.TimestampFormat, v)
 			assert.Nil(t, err, "Build time format incorrect")
-			assert.WithinDuration(t, time.Now(), buildTime, time.Second*1)
+			assert.WithinDuration(t, time.Now(), buildTime, time.Minute*1)
 		} else {
 			actualEnv[entry] = true
 		}

